File: programmers/js_src/level00/120889.js
// 삼각형의 완성조건 (1)
function solution(sides) {
  var answer = 0;

  sides.sort((a, b) => b - a);

  if (sides[1] + sides[2] > sides[0]) {
    answer = 1;
  } else {
    answer = 2;
  }

  return answer;
}

File: programmers/java/level00/120889.java
// 삼각형의 완성조건 (1)

import java.util.Arrays;

class Solution {
    public int solution(int[] sides) {
        Arrays.sort(sides);
        
        return sides[0] + sides[1] > sides[2] ? 1 : 2;
    }
}